lib/AACStream/aacs.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * mux.js
+ *
+ * Copyright (c) 2016 Brightcove
+ * All rights reserved.
+ *
+ * A stream-based aac to mp4 converter. This utility can be used to
+ * deliver mp4s to a SourceBuffer on platforms that support native
+ * Media Source Extensions.
+ */
+'use strict';
+var Stream = require('../utils/stream.js');
+
+var
```

Comment:
Either everything below this needs to be indented differently or you should declare things here and then define them below.